jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -125,7 +126,8 @@ ecma_builtin_global_object_parse_int (ecma_value_t this_arg __attr_unused___, /*
   ecma_length_t end = str_size;
   for (ecma_length_t i = 0; i < end; i++)
   {
-    if (!(isspace (utf8_string_buff[i])))
+    if (!lit_char_is_white_space (utf8_string_buff[i])
+        && !lit_char_is_line_terminator (utf8_string_buff[i]))
```

Comment:
@zherczeg, yes, you are right. The code doesn't fully support unicode yet.

Purpose of the pull request is only to remove `isspace`-like functions and switch to `lit_char_*` helpers. May be, it is too early to perform the change.

What do you think about this?